ajax/libs/tent-css/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tent-css",
+  "filename": "tent.min.css",
+  "description": "A CSS survival kit.",
+  "author": "Aaron Mazade <ulinaaron@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sitetent/tentcss.git"
+  },
+  "homepage": "https://css.sitetent.com",
+  "keywords": [
+    "css",
+    "flexbox",
+    "html",
+    "html5",
+    "scss",
+    "sass",
+    "responsive",
+    "framework"
+  ],
+  "npmName": "tent-css",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.css"
```

Comment:
Let's prevent to only fetch `css` files because `basePath` is `dist` which is normally all files users need. Instead try to exclude `_forms.control.css` & `_forms.inputs.css` under `06_components` in v1.1.0 ~ v1.2.1. Thanks!